src/modules/drawing/components/DrawingModifyInteraction.vue

```diff
@@ -32,13 +33,18 @@ export default {
             // here we can't have the whole drawing layers features (or source)
             // otherwise non-selected features will be editable anytime
             features: this.getSelectInteraction().getFeatures(),
-            style: editingFeatureStyleFunction,
+            style: editingVertexStyleFunction,
             deleteCondition: (event) => noModifierKeys(event) && singleClick(event),
             // This enables click on the shape of features (instead of pixel tolerance from their coordinates)
-            hitDetection: true,
+            // commented for now is it has performance issues with geodesic lines
+            //hitDetection: true,
```

Comment:
The problem only raises when you draw a long line with many points (e.g. a 9500 km long line, as it is long and is internally represented with nearly 1000 points) and then zoom near, click the feature and hover with the mouse over it. But then there is actually a second place in the code that also causes the same performance problem (line 97 in DrawingTooltip.vue which is responsible for updating the tooltip on pointer move) so commenting out "hitDetection: true" is actually not enough to solve the performance problem in this case.

But enabling hitDetection causes the markers and text fields to not be movable at all after saving the drawing and reloading the page. This bug was already (at least partially) present before this PR. So that's why I would still left it deactivated. I will update the comment in the code accordingly.